src/content/certifications/exam-faqs/vault-operations-professional.mdx

```diff
@@ -1,102 +1,83 @@
 <!-- Each "## Heading Two" section in this document will be used to populate an FAQ item -->
 
-## Prerequisites
+## Who should take this exam 
 
-We strongly recommend passing the associate-level Vault exam before taking the professional-level exam. Practitioners who are already experienced with Vault operations in a production environment—and understand the concepts covered in the associate exam— may be able to successfully pass the professional-level exam.
+You should take the Vault Operations Professional exam if you are a Cloud engineer with advanced, production-level Vault operations expertise. You will need to demonstrate your ability to deploy, configure, manage, and monitor HashiCorp Vault, and you must also be able to evaluate Vault Enterprise functionality and use cases. 
 
-- HashiCorp Certified: Vault Associate Certification (recommended)
-- Linux skills such as list and edit files via command terminal
-- Understanding of IP networking
-- Experience with Public Key Infrastructure (PKI), including PGP and TLS
-- Information security fundamentals such as network security and RBAC
-- Understand the concepts and functionality of infrastructure running in containers including starting and stopping services, and reading logs
+Prerequisites:
 
-<ThemedImage
-	lightSrc="https://www.datocms-assets.com/2885/1680025449-vault_cert_lt-1.png"
-	darkSrc="https://www.datocms-assets.com/2885/1680025429-vault_cert_dt-1.png"
-	alt=""
-	width="515"
-	height="279"
-/>
+- Vault Associate certification (strongly recommend, or equivalent experience required) 
+- Linux skills such as list and edit files via command terminal 
+- Understanding of IP networking 
+- Experience with Public Key Infrastructure (PKI), including PGP and TLS 
+- Information security fundamentals such as network security and RBAC 
+- Understand the concepts and functionality of infrastructure running in containers including starting and stopping services, and reading logs 
 
-## Exam Details
+## Exam details
 
 |                     |                                                                                                                                                        |
 | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| **Assessment Type** | Lab-based and multiple choice                                                                                                                          |
+| **Assessment Type** | [Lab scenarios](vault/tutorials/ops-pro-cert/ops-pro-overview) and multiple choice                                                                                                                          |
```

Comment:
This link is broken. It appends the suffix you put to the wrong prefix. I think it because where we are and where you are linking to don't share a prefix. You maybe have to do an absolute path instead of relative but ask someone knows this stuff if there is a better way.